interfaces/builtin/utils_test.go

```diff
@@ -89,3 +89,16 @@ func MockConnectedSlot(c *C, yaml string, si *snap.SideInfo, slotName string) (*
 	}
 	panic(fmt.Sprintf("cannot find slot %q in snap %q", slotName, info.InstanceName()))
 }
+
+func MockHotplugSlot(c *C, yaml string, si *snap.SideInfo, hotplugKey, ifaceName, slotName string, staticAttrs map[string]interface{}) *snap.SlotInfo {
+	info := snaptest.MockInfo(c, yaml, si)
```

Comment:
Could this become:
```
slotInfo := MockSlot(c, yams, si, slotName)
slotInfo.Attrs = staticAttrs
slotInfo.HotplugKey = hotplugKey
return slotInfo
```

Reply:
No, because `MockSlot` requires the slot to be decleared in the yaml, this is exactly what I don't want here.